### ISSUE TITLE
Rails 6.1 support

### ISSUE THREAD

**Issue by nukah** [rkrage/pg_party#60]
Are you planning to change specification in Gemfile to allow activerecord >= 6?

**Comment by rkrage** [comment]
My intention was to support new versions of Rails after updating the build matrix and doing some manual testing to confirm things still work (this gem relies on a handful of internal / private APIs). It _is_ a little annoying that I need to update the gemspec and release a new version of this gem for every Rails release. What if I opened things up to `"activerecord", ">= 5.0", "< 7"` and just add a note in the readme about which Rails versions the gem is tested against?

**Comment by nukah** [comment]
That's an option as well. It really depends on conflicts and errors with every new activerecord/rails release.

**Comment by darash** [comment]
rails 6..1.1

```
Bundler could not find compatible versions for gem "activerecord":
  In Gemfile:
    pg_party x86_64-linux was resolved to 1.4.0, which depends on
      activerecord (>= 5.0, <= 6.1)

    rails (>= 6.1.1) x86_64-linux was resolved to 6.1.1, which depends on
      activerecord (= 6.1.1)
```
